istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -581,6 +583,9 @@ func workloadConfigCmd(ctx cli.Context) *cobra.Command {
   # Retrieve workload summary
   kubectl exec -it $ZTUNNEL -n istio-system -- curl localhost:15000/config_dump > ztunnel-config.json
   istioctl proxy-config workloads --file ztunnel-config.json
+
+  # Retrieve workload summary for a specific namespace
+  istioctl proxy-config workloads <ztunnel-name[.namespace]> --config-namespace foo
```

Comment:
```suggestion
  istioctl proxy-config workloads <ztunnel-name[.namespace]> --workload-namespace foo
```

nit: `config-namespace` is a bit confusing/non-obvious/non-intuitive as a flag name.

- `--workload-namespace`
- `--namespace` 

would be other options

Reply:
`--workload-namespace` does make more sense to me. I prefer it over the other option as we support both `<ztunnel-name[.namespace]>` and `<ztunnel-name> --namespace` as the ztunnel name input.